src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -58,6 +58,7 @@ class PhpDumper extends Dumper
     private $targetDirRegex;
     private $targetDirMaxMatches;
     private $docStar;
+    private $existingNames = array();
```

Comment:
It seems to describe the purpose of the variable with absolute precision, so it's probably a good idea to make that change ... and, after that change, it's probably a good idea to change `$uniqueName` to `$methodName` or `$serviceMethodName` too.